data/hai/hai reveal 1 intro.txt

```diff
@@ -0,0 +1,1002 @@
+				decline
+
+#This gets picked up silently on Hai-home when landing for response 2 - leads to go kill pirates mission with chat on Allhome.
+mission "Hai Leaks Response 3"
+	name "Meet with Security Forces on Allhome"
+	description "Meet with the Security Forces on <destination> to fight pirates."
+	landing
+	source "Hai-home"
+	destination "Allhome"
+	to offer
+		has "Hai Leaks Response 2: done"
+
+#Go kill pirates mission with chat on Allhome.
+mission "Hai Leaks Response 4"
+	name "Defeat the Black Lich"
+	description "Defeat Black Lich's Marauder Leviathan at <waypoints>, then return to <origin>."
+	autosave
+	landing
+	source "Allhome"
+	waypoint "Waypoint"
+	destination "Allhome"
+	to offer
+		has "Hai Leaks Response 3: done"
+	on offer
+		conversation
+			`When you land on Allhome, you are greeted by the Hai Captain of the local forces. You're not an expert at judging Hai ages, but he appears to be a little younger than expected.`
+			`	"Captain <last>! I heard you were coming to assist."`
+			choice
+				`	"Time to route some pirates baby!"`
+				`	"I'm here to reinforce you. What can I do?"`
+			`	"Uh well, yes. I presume you saw the pirates on your way in? They've been appearing en masse and overwhelming security forces to raid technology and goods from the Hai traders and their security. Some of them have even been sneaking down onto the planet, capturing Hai, and taking them away."`
+			choice
+				`	"I have heard about the kidnapping."`
+			`	He nods. "Unfortunately, it's taking a while for them to redistribute an appropriately resourced defense force, thanks to the Unfettered. It's bad enough that we've had to bring the old Ladybug models out of mothballs to bolster our forces here."`
+			`	"Even more unfortunately," he continues, "it seems that the overabundance of these pirates is the result of one particular individual's success. His ship, called the Black Lich, is in Waypoint with a stolen Geocoris that has been converted into some kind of semi-stable weapons platform and is now being used as a temporary base."`
+			`	"This does give us an opportunity, however. If the kidnapped Hai are still on board, we might be able to rescue them, and if we can defeat the Black Lich, we may stem the tide of opportunists that are swarming through the wormhole.`
+			`	"What do you say?"`
+			`	The poor Captain seems desperate.`
+			choice
+				`	"We'll capture the geocoris and bring them safely home."`
+					accept
+	npc
+		government "Hai"
+		personality escort heroic disables
+		fleet "Anti-Pirate Hai (No Wormhole Travel)" 5
+	npc
+		government "Pirate"
+		personality staying disables plunders harvests
+		system "Waypoint"
+		fleet "Marauder fleet IX" 4
+		fleet "Hired Guns" 1
+	npc kill
+		government "Pirate"
+		personality staying marked target
+		system "Waypoint"
+		ship "Marauder Leviathan (Weapons)" "Black Lich"
+	to complete
+		or
+			has "00 Hai Leaks Response 4B: declined"
+			has "Hai Leaks Response 4A: failed"
```

Comment:
I'm pretty sure in its current state you could get softlocked out of the story here if you capture the Geocoris but lose it before landing, since the completion conditions are you either blew up the Geocoris before capturing it or you successfully landed with the Geocoris as your flagship.

Reply:
Erm, you sure-sure? I seem to recall a decent effort to prevent that during a round of testing last year.

Reply:
"00 Hai Leaks Response 4B: declined" is only set if you successfully captured the Geocoris and landed with it as a flagship.
"Hai Leaks Response 4A: failed" is only set if you failed to capture the Geocoris.
Neither of these conditions capture the scenario where you successfully capture the Geocoris but fail to land with it as a flagship (e.g. it was destroyed after capture or, God forbid, you sold it).

Reply:
If I recall correctly, one of the missions is meant to check that you have the outfit that the Geocoris carries, and if you do proceed and cancel the other mission as a get out of that loop situation.